src/main/java/frc/robot/subsystems/drive/SwerveModule.java

```diff
@@ -45,6 +46,8 @@ public SwerveModule(int drivingCanId, int turningCanId, double chassisAngularOff
     turningSparkMax = new CANSparkMax(turningCanId, MotorType.kBrushless);
     chassisAngularOffsetRadians = chassisAngularOffset;
 
+    SparkMaxUtils.initWithRetry(this::initDriveSpark, Calibrations.SPARK_INIT_RETRY_ATTEMPTS);
```

Comment:
Make an initialize function and call in there. Also call for the turnspark

Reply:
Also add a swerve system initialize function that call the initialize for all our modules